content/pages/tools/markdoc.html

```diff
@@ -0,0 +1,139 @@
+       onclick="jQuery('#signup-modal').modal('show');">
+        <i class="fad fa-play icon"></i>
+        Create an account
+      </a>
+      <a href="https://example-markdoc.readthedocs.io/en/latest/" class="ui large stackable button">
+        <i class="fad fa-camera-retro icon"></i>
+        See a demo
+      </a>
+    </p>
+  {% endcall %}
+</section>
+
+<section id="features">
+  {% call marketing.highlight_list() %}
+
+    {% call marketing.highlight_item(
+        icon="olive fa-people-arrows",
+        icon_style="--fa-secondary-color: violet; --fa-secondary-opacity: 1;",
+        header="Easily share with deploy previews",
+        color="violet") %}
+      {% markdown %}
+      Receive feedback from your team,
+      and <strong>have confidence in changes</strong> before they go live.
+      {% endmarkdown %}
+    {% endcall %}
+
+    {% call marketing.highlight_item(
+        icon="yellow fa-magnifying-glass",
+        icon_style="--fa-secondary-color: white; --fa-secondary-opacity: 1;",
+        header="Always find what you need",
+        color="violet") %}
+      {% markdown %}
+      Search across all your projects with our search-as-you-type interface,
+      and <strong>never lose documentation again</strong>.
+      {% endmarkdown %}
+    {% endcall %}
+
+    {% call marketing.highlight_item(
+        icon="fa-eye",
+        header="Keep your focus with link previews",
+        color="violet") %}
+
+    {% markdown %}
+    Provide context on every internal link in your docs with instant preview links.
+    <strong>Skim the content without opening *another* tab</strong>.
+    {% endmarkdown %}
+    {% endcall %}
+
+  {% endcall %}
+</section>
+
+{# Try it out code block #}
+{% include "includes/try-it-out-markdoc.html" %}
+
+{# Markdoc user testimonials #}
+{# {% include "includes/users-markdoc.html" %} #}
+
+{% block homepage_bottom_callout %}
+<section>
+  <div class="ui very padded container">
+    <div class="ui grid center aligned">
+
+      <div class="row">
+        <div class="column twelve wide computer sixteen wide tablet">
+
+          {#
+            TODO replace this block with something more actionable. It's
+            not saying anything new at this point in the page, and the call
+            to action is not strong.
+          #}
+          <div class="ui basic vertical huge segment">
+            <p>
+              <b>Import your Markdoc project today.</b>
+            </p>
+
+            <p>
+              <a class="ui large teal stackable button"
+              data-analytics="signup-modal"
+               onclick="jQuery('#signup-modal').modal('show');">
+              <i class="fad fa-play icon"></i>
+              Create an account
+            </a>
+            </p>
+          </div>
+
+          <p>
+            Learn more about using Markdoc with Read the Docs in our <a href="https://docs.readthedocs.io/en/stable/intro/markdoc.html">documentation</a>.
+          </p>
```

Comment:
This last paragraph doesn't render nicely in the page. It would be good if we find another place for it.